src/object.c

```diff
@@ -187,6 +199,7 @@ V7_PRIVATE void init_object(struct v7 *v7) {
   object = v7_get(v7, v7->global_object, "Object", 6);
   v7_set(v7, object, "prototype", 9, v7->object_prototype);
 
+  object = v7->object_prototype;
```

Comment:
This is incorrect. The following properties are defined on `Object` not on `Object.prototype` (except `isPrototypeOf` see below).